s3/pkg/utils/main/mainencdec.go

```diff
@@ -0,0 +1,57 @@
+package main
```

Comment:
Is this file a test utility? required?

Reply:
main file, for quick testing of encryption and decryption